src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIResponsesChatClient.cs

```diff
@@ -162,6 +162,11 @@ internal static ChatResponse FromOpenAIResponse(OpenAIResponse openAIResponse, R
             }
         }
 
+        foreach (var message in response.Messages)
+        {
+            message.CreatedAt = openAIResponse.CreatedAt;
```

Comment:
Would similar changes be required in externally defined clients like the one in OllamaSharp?

Reply:
Desirable, probably. Required, no.